tests/Application/config/packages/twig.yaml

```diff
@@ -2,3 +2,11 @@ twig:
     paths: ['%kernel.project_dir%/templates']
     debug: '%kernel.debug%'
     strict_variables: '%kernel.debug%'
+
+services:
+    _defaults:
+        public: false
+        autowire: true
```

Comment:
Could you check, if it'd needed?

Reply:
I think we can leave it like that - the same way it is done in the `sylius/pluginskeleton` and in our other plugins

Reply:
It is not necessary